src/main/java/com/training/sample/pages/SamplePage.java

```diff
@@ -1,8 +1,15 @@
 package com.training.sample.pages;
```

Comment:
Since this is a utility of this framework, This test should be placed in - test/java/com/training/utilities/tests/
and all the implementation should go in - main/java/com/training/utilities/